static/app/utils/hotkeyslistener.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
we need a lint rule for this, but prefer importing `{useRef, useCallback, useState}` over just `React`.

It will help with codesplitting in the future.